src/get-shard-mapping.js

```diff
@@ -0,0 +1,31 @@
+const utils = require('./utils');
+
+const get_shard_mapping = async () => {
```

Comment:
Please never use snake case, and always use camel case for function names.

Reply:
PEP8 habits die hard :)